### ISSUE TITLE
QIntersection "hurts" SSR

### ISSUE THREAD

**Issue by eladcandroid** [quasarframework/quasar#6486]
I'm using the lovely q-intersection with prefetched items but I can't see them on the page view-source.. when I'm just replacing q-intersection element into div, the items appear again... 

Example code:
```javascript
      <q-intersection
        once
        v-for="item in items"
        :key="item.slug"
        class="col-md-3 col-sm-4 col-xs-12 item-card"
      > .... </q-intersection>
```

    Dev mode.......... ssr
    Pkg quasar........ v1.9.4
    Pkg @quasar/app... v1.5.8

**Comment by pdanpdan** [comment]
QIntersection only shows the content when it's visible in the client (browser) viewport, so there is no way to use it with SSR as it is now.

**Comment by eladcandroid** [comment]
@pdanpdan Thanks. Now i'm handling this with q-no-ssr with a placeholder. 

```javascript
<q-no-ssr class="row justify-center items q-gutter-md q-ma-md">
        <q-intersection
          once
          v-for="item in items"
          :key="item.slug"
          class="col-md-3 col-sm-4 col-xs-12 item-card"
        >
          <ItemPreview :item="item" />
        </q-intersection>
        <template v-slot:placeholder>
          <div
            once
            v-for="item in itemsPaginated"
            :key="item.slug"
            class="col-md-3 col-sm-4 col-xs-12 item-card"
          >
            <ItemPreview :item="item" />
          </div>
        </template>
</q-no-ssr>
```

If there's a better way, please let me know.

**Comment by pdanpdan** [comment]
Your solution is very good (I suppose itemsPaginated is a list of above the fold items).
I'll try to find a better solution if I can, so please leave the issue open to track it.

Thank you

**Comment by pdanpdan** [comment]
@rstoenescu Do you think a `on-ssr` prop that would force render on SSR would be a good approach?
It would allow pre-rendering a limited number of items (the above the fold part).

**Comment by rstoenescu** [comment]
If the developer wants to show something regardless of the Intersection observer (which obviously can't run on server-side) then he/she shouldn't use QIntersection for the first N items to begin with. If we add a prop to circumvent the target behavior, then why is the component needed in the first place?

**Comment by rstoenescu** [comment]
After some more discussions --> new Boolean prop "ssr-prerender" will be available in "quasar" v1.9.5.
